resources/profiles/Creality.json

```diff
@@ -78,6 +78,10 @@
 			"name": "fdm_process_creality_common",
 			"sub_path": "process/fdm_process_creality_common.json"
 		},
+		{
+			"name": "fdm_process_common_klipper",
+			"sub_path": "process/fdm_process_common_klipper.json"
```

Comment:
Sweet, thanks for adding the inheritance node! I'll make a follow-up PR in the future to consolidate more into this like acceleration and percentage-based line widths.

Can you call this `fdm_process_common_klipper_corexy`? Creality's starting to put Klipper on their bed slingers like the Ender 3 V3 KE and we'd probably want much lower acceleration values there eventually.

Reply:
I didn't really pay attention to any of their product.
But in the future, if there are different serial, they can inherit this file and make certain changes there.

Reply:
Sounds good.